docs/notebooks/zero_inflated_regression.ipynb

```diff
@@ -0,0 +1,1905 @@
+{
```

Comment:
<p>The warning is telling that the LOO-CV approximation may not be reliable, so we shouldn't use it to perform model comparison. I think I would remove this part, even though one can feel tempted to perform it.</p><p> </p><p>Instead, I would have a section where we reflect on the different data generating processes behind the two distributions. I <em>believe</em> this dataset is more reasonable for a ZIPoisson since the zeroes can appear because the group doesn't fish, or because they fished but they caught zero fish. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>